ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -524,15 +469,16 @@ ColumnLayout {
 
             width: 520
             destroyOnClose: true
-            confirmButtonLabel: qsTr("Hide all collectibles minted by this community")
+            confirmButtonLabel: qsTr("Hide '%1' collectibles").arg(communityName)
```

Comment:
Did this design changed as well??

Reply:
Yeah, it was always overflowing

Reply:
Can you share the design? Something similar for assets?